src/AWS/CRT/CRT.php

```diff
@@ -22,14 +22,15 @@ function __construct() {
         if (is_null(self::$impl)) {
             // Figure out what backends are/should be available
             $backends = ['Extension'];
-            if (version_compare(PHP_VERSION, '7.0.0') >= 0) {
-                $backends = ['Extension', 'FFI'];
-                if (getenv('AWS_CRT_PHP_EXTENSION')) {
-                    $backends = ['Extension'];
-                } else if (getenv('AWS_CRT_PHP_FFI')) {
-                    $backends = ['FFI'];
-                }
-            }
+            // Disabled until FFI is fully supported/works properly
```

Comment:
Maybe remove if we aren't planning on using FFI any time soon?

Reply:
Yeah, I can pull the FFI stuff out